### ISSUE TITLE
when applying or removing a license from a project, never start or stop or restart the project, unless it is already currently running

### ISSUE THREAD

**Issue by williamstein** [sagemathinc/cocalc#5667]
For example, this is what happens right now:

1. Find an archived project you have.
2. Apply a license to it.
3. Watch the project unarchive and start, even though you don't click "Start project".

**Comment by williamstein** [comment]
This annoys me, but I'm also marking it low priority, since it's very unlikely to actually annoy anybody else, as odds are that if you're in a project's settings, you probably want to use that project...
